doc/build-windows.md

```diff
@@ -9,7 +9,7 @@ The options known to work for building Dash Core on Windows are:
 and is the platform used to build the Dash Core Windows release binaries.
 * On Windows, using [Windows
 Subsystem for Linux (WSL)](https://docs.microsoft.com/windows/wsl/about) and the Mingw-w64 cross compiler tool chain.
-* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com).
+* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com). See [README.md](/build_msvc/README.md).
```

Comment:
drop 19061

Reply:
Maybe keep it but add a note that it's incomplete?

Reply:
What do you mean? We don't have build_msvc yet so we shouldn't do this backport no?

Reply:
oh, right 🙈 drop it then :)